java/server/test/org/openqa/grid/internal/ProxySetTest.java

```diff
@@ -58,77 +53,4 @@ public void removeIfPresent() {
       registry.stop();
     }
   }
-
-  @Test
-  public void testProxySortingByIdle() throws Exception {
```

Comment:
removing tests and not adding one is not beneficial here. Please add a test, otherwise I'm not sure the premise of your bug #1673 is correct.

Also, since this test existed and was passing, not sure what was wrong with this chunk of code.